include/build_config/SDL_build_config_wingdk.h

```diff
@@ -75,6 +75,7 @@
 #define HAVE_LIBC   1
 #define HAVE_MALLOC 1
 #define HAVE_CALLOC 1
+#define HAVE_FDATASYNC 1
```

Comment:
Are we sure about platform config changes? E.g.: Does everyone really have fdatasync ?

Reply:
On ci, only Haiku doesn't have it (it uses `fsync` instead)
Off ci, I don't know

Reply:
Don't know haiku, but windows & variants and emscripten look suspicious.

Reply:
.. and making sure about macos, ios, and android would be best..